File: models/JobLanguage.js
const Sequelize = require('sequelize');
const db = require("../config/db");
const Job = require("./Job");
const Language = require("./Language");
const LanguageLevel = require("./LanguageLevel");

const JobLanguage = db.define("JobLanguage", {
    JobId: {
        type: Sequelize.INTEGER,
        allowNull: false
    },
    LanguageCode: {
        type: Sequelize.CHAR(2),
        allowNull: false 
    },
    LanguageLevelCode: {
        type: Sequelize.CHAR(2),
        allowNull: false
    }
},
{
    timestamps: false,
    freezeTableName: true
});

Language.hasMany(JobLanguage);
JobLanguage.belongsTo(Language);

module.exports = JobLanguage;

File: API/Models/JobLanguage.cs
using System;
using System.Collections.Generic;

namespace JobberAPI.Models;

public partial class JobLanguage
{
    public int Id { get; set; }

    public int JobId { get; set; }

    public string LanguageLevelCode { get; set; } = null!;

    public string LanguageCode { get; set; } = null!;

    public virtual Job Job { get; set; } = null!;

    public virtual Language LanguageCodeNavigation { get; set; } = null!;

    public virtual LanguageLevel LanguageLevelCodeNavigation { get; set; } = null!;
}
